apps/dolly-backend/src/main/java/no/nav/dolly/bestilling/sykemelding/dto/NySykemeldingResponse.java

```diff
@@ -0,0 +1,17 @@
+package no.nav.dolly.bestilling.sykemelding.dto;
+
+import java.time.LocalDate;
+import java.util.List;
+
+public record NySykemeldingResponse(
```

Comment:
Hvorfor er request og response i to ulike pakker, burde de ikke være i samme?

Reply:
Nei, er på gyngende grunn selv, disse har jo ganske lik bruk.
For meg funger det å ha de i domain, det er jo en domain body i request og response kan jo forsvares at det er et data transfer object (DTO), i så fall bør disse konsekvent slutte på DTO, ref arven fra Martin. Men, man kan også hevde at dette er domain (respons domain) og sett forsvare plassering der. 
DTOer brukes jo for intern transport mellom klasser. 😄